tests/framework/clients/rest_client.go

```diff
@@ -75,7 +71,21 @@ func CreateRestAPIClient(platform enums.RookPlatformType, k8sHelper *utils.K8sHe
 	}
 	client := rclient.NewRookNetworkRestClient(endpoint, httpclient)
 
-	return &RestAPIClient{client}
+	//make sure rest client is up and available
+	inc := 0
+	for inc < 20 {
```

Comment:
Aha, I thought I was writing to the log,  guess I missed it, Should be fixed now.
Also adding cost for retry variables.